japfohl/cs360-projectb-group20
# Landing results ui

This PR incorporates a basic rendering framework for the site.  It uses a set of templates to render a given page and allows for max flexibility and max code reuse.  Here are some recommended steps for testing:

Notes: This build assumes that you've run the DB creation code and populated the DB with values.

### To Test:
1. Do a `git pull` to ensure that you've got the code from this branch.
2. Do a `git checkout landing-results-ui` to jump onto this branch.
3. Click the "run project" button @ the top of the C9 work space. 
4. Once Apache is up and running, open the link in a new browser window / tab. 
     * You should be automatically redirected to the home page (*which is admittedly sparse at this point*).
5. Search for a term you know is in the DB (honey or refrigerator are good examples)
     * You should be send to a results.php page where the results of your search are displayed.
6. Verify that everything renders and the search results work as expected.

At this point, the focus of this PR was to get the logic up and running, build an easy to use framework, and ensure that we could query the DB for user supplied terms.  Future work can focus on making things pretty.

As always, comment with questions, thoughts, concerns, bugs or otherwise and Steve and I can jump on them.
